documentation/src/main/asciidoc/reference/mapper-orm-bridge-basics.asciidoc

```diff
@@ -66,7 +66,7 @@ Value fields as well as <<mapper-orm-bridge-index-field-dsl-object,object fields
 |`@PropertyBinding`
 |`@TypeBinding`
 |<<mapper-orm-identifiermapping,`@DocumentId`>>
-|`@Indexed(routingBinder = ...)`
+|`@Indexed( routingBinder = ... )`
```

Comment:
Shall we add a space after the parenthesis?

Reply:
After the closing one? I don't see why we would.

I added these spaces to reduce a bit the width of the cell, that's all.

Reply:
I meant after the opening one, anyway it's OK

Reply:
Then that's exactly what I did, and I don't understand the question :) Maybe I already answered.